src/engine/source/indexerconnector/src/indexerConnector.cpp

```diff
@@ -221,13 +257,17 @@ IndexerConnector::IndexerConnector(const IndexerConnectorOptions& indexerConnect
                 if (parsedData.at("operation").get_ref<const std::string&>().compare("DELETED") == 0)
                 {
                     const auto& id = parsedData.at("id").get_ref<const std::string&>();
-                    builderBulkDelete(bulkData, id, m_indexName);
+                    bulkData += IndexerQueryBuilder::builder().deleteIndex(m_indexName, id).build();
```

Comment:
Now we can remove `builderBulkDelete()` ?

Reply:
Done!